data/assets/examples/renderable/renderableorbitdisc/renderableorbitdisc.asset

```diff
@@ -0,0 +1,22 @@
+local OrbitDisc = {
+    Identifier = "RenderableOrbitDisc_Example",
+    Renderable = {
+      Type = "RenderableOrbitDisc",
+      Eccentricity = 0.2,
+      Size = 10000,
+      Offset = { 0.1, 0.7 },
+      Texture = asset.resource("texture.png"),
```

Comment:
I uploaded the texture to data.openspaceproject.com/examples/barber_pole.png.  I think in general we should prevent having binary files in the folders as much as possible.  In this case, it's only 137 bytes, but still?